source/guides/clear/mixer.rst

```diff
@@ -411,27 +479,197 @@ mix version 10 (from which the image was built), to mix version 20.
       swupd update
       swupd bundle-list -a
 
-#. Now your mix should be at version 20 and curl is now available. Try using
+#. Now your mix should be at version 20 and curl is available. Try using
    curl. This will fail because curl is not yet installed:
 
    .. code-block:: console
 
       curl: command not found
       To install curl use: swupd bundle-add curl
 
-   Add the new bundle from your update server to your VM. Retry curl. It works!
+#. Add the new bundle from your update server to your VM. Retry curl.
+   It works!
 
    .. code-block:: bash
 
       swupd bundle-add curl
       curl -O https://download.clearlinux.org/image/start_qemu.sh
 
-   Shutdown your VM:
+#. Shutdown your VM:
 
    .. code-block:: bash
 
       poweroff
 
+Example 6: Remove a bundle
+==========================
+
+Removing a bundle requires more work than deleting the bundle description
+file, as shown in Example 2. After a bundle is built in the mix, you must
+assure all of the files that are part of the bundle are removed from the
+client where that bundle is installed. To do this, create a version of this
+bundle in which all of its content is marked for deletion.
+
+In the following example, we show how to remove the contents of the `editors`
+bundle that we added to our mix in Example 2.
+
+#. First update your mix version. This will set the mix to the next version.
+
+   .. code-block:: bash
+
+      mixer versions update
+
+   .. note::
+      Run this command every time that you want to build a new version.
+
+#. Navigate to local-bundles:
```

Comment:
I would collapse lines 525-548 to simply this:

Edit the local `editors` bundle definition and delete all lines after the [MAINTAINERS] line.

.. code-block:: bash

$EDITOR ./local-bundles/editors

It should look like this:

.. code-block:: bash

# [TITLE]: editors
# [DESCRIPTION]: Run popular terminal text editors.
# [STATUS]: Active
# [CAPABILITIES]:
# [TAGS]: Tools and Utilities, Editor
# [MAINTAINER]: Developer <developer@intel.com>